includes/multi-currency/MultiCurrency.php

```diff
@@ -1019,6 +1019,8 @@ public function display_geolocation_currency_update_notice() {
 		$notice_id = md5( $message );
 
 		echo '<p class="woocommerce-store-notice demo_store" data-notice-id="' . esc_attr( $notice_id . 2 ) . '" style="display:none;">';
+		// No need to escape here as the function called handles it.
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 		echo \WC_Payments_Utils::esc_interpolated_html(
 			$message,
```

Comment:
Thank you @reykjalin, good catch there and thanks for looking into it. I wasn't aware of that rule.

Reply:
Looks like this is actually an issue with the ruleset and not necessarily our config (although I think the config is wrong as well):

See https://github.com/WordPress/WordPress-Coding-Standards/issues/1176 and https://github.com/WordPress/WordPress-Coding-Standards/pull/2370.

I think the config is wrong based on [this example](https://github.com/WordPress/WordPress-Coding-Standards/wiki/Customizable-sniff-properties/_compare/6a276c6762f1be8fd8a14a5fb9090e54889cc8bc...2798f6cb684918364e1dec6a1c6d85a2c1fbed77).

Either way, I guess this can wait until the next time we update the PHPCS sniffs since all those issues and PRs are still open 🫠